examples/ml-pipelines-node-classification.ipynb

```diff
@@ -553,124 +490,14 @@
-       "      <th>1</th>\n",
-       "      <td>1</td>\n",
-       "      <td>5</td>\n",
-       "      <td>[0.10248697035946842, 0.04786333496925834, 0.1...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>2</th>\n",
-       "      <td>2</td>\n",
-       "      <td>2</td>\n",
-       "      <td>[0.04005138780445344, 0.4716897821336281, 0.04...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>3</th>\n",
-       "      <td>3</td>\n",
-       "      <td>2</td>\n",
-       "      <td>[0.03168648772215199, 0.709948522271949, 0.119...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>4</th>\n",
-       "      <td>4</td>\n",
-       "      <td>3</td>\n",
-       "      <td>[0.05023837133685246, 0.022112223083451652, 0....</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>5</th>\n",
-       "      <td>5</td>\n",
-       "      <td>5</td>\n",
-       "      <td>[0.163967792875254, 0.17289763778273642, 0.108...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>6</th>\n",
-       "      <td>6</td>\n",
-       "      <td>6</td>\n",
-       "      <td>[0.055509897709602456, 0.11420633716452112, 0....</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>7</th>\n",
-       "      <td>7</td>\n",
-       "      <td>100</td>\n",
-       "      <td>[0.0357353343079603, 0.06031988891476322, 0.05...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>8</th>\n",
-       "      <td>8</td>\n",
-       "      <td>100</td>\n",
-       "      <td>[0.07296843717126494, 0.09662250045306757, 0.2...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>9</th>\n",
-       "      <td>9</td>\n",
-       "      <td>4</td>\n",
-       "      <td>[0.056177601249482755, 0.16500456321738488, 0....</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "   nodeId  predictedClass                             predictedProbabilities\n",
-       "0       0             100  [0.07090129256540621, 0.02781562683066478, 0.2...\n",
-       "1       1               5  [0.10248697035946842, 0.04786333496925834, 0.1...\n",
-       "2       2               2  [0.04005138780445344, 0.4716897821336281, 0.04...\n",
-       "3       3               2  [0.03168648772215199, 0.709948522271949, 0.119...\n",
-       "4       4               3  [0.05023837133685246, 0.022112223083451652, 0....\n",
-       "5       5               5  [0.163967792875254, 0.17289763778273642, 0.108...\n",
-       "6       6               6  [0.055509897709602456, 0.11420633716452112, 0....\n",
-       "7       7             100  [0.0357353343079603, 0.06031988891476322, 0.05...\n",
-       "8       8             100  [0.07296843717126494, 0.09662250045306757, 0.2...\n",
-       "9       9               4  [0.056177601249482755, 0.16500456321738488, 0...."
```

Comment:
I believe @nvitucci argued to keep some of the cell outputs for documentational purposes. Personally, I'm neutral.

Reply:
Yes, my suggestions was to keep some output for two reasons:
- documentation purposes (see e.g. the example output on https://neo4j.com/docs/graph-data-science-client/current/tutorials/load-data-via-graph-construction/#_load_the_cora_dataset)
- (potentially) testing purposes, i.e. to make sure that a result stay consistent (as an alternative to explicit `asserts`).

If you can leave this untouched for now, so it keeps reflecting the published docs, we can have a separate discussion on it afterwards.